docs/userman/gui/pages/index.md

```diff
@@ -282,3 +282,204 @@ runs.
     This application does the same as in the previous example, except that you now
     have the footer line (*'This application was created...'*) in all the pages of
     your application.
+
+# Page modules {data-source="gui/examples/grocery_store"}
+
+In Taipy, a Page represents a specific part of the whole web interface. While it is possible to
+define a single-page application directly within the main script, real-world applications typically
+require multiple pages to structure different views and functionalities.
+
+To improve maintainability and modularity, it is a good practice to define each page in a specific
+Python module where each individual page is defined. We call such a module a **Page Module**,
+because it is an actual Python module and because it defines... a page.<br/>
+Modules are typically grouped within a dedicated directory (e.g., `pages/`) and imported into the
+main script. This approach enhances code organization, making it easier to manage, update, and
+scale an application.
+
+Page Modules provide benefits in different areas:
+
+- **Code Organization**: Keeping page definitions in separate files avoids clutter in the main
```

Comment:
avoids here is wrong imo. should be either `to avoid` or just `avoid` depending on the noun usage.